.circleci/config.yml

```diff
@@ -12,28 +11,44 @@ executors:
       image: "ubuntu-2004:202201-02"
     environment:
       COMPOSE_PROJECT_NAME: "circleci"
+  ubuntu_xlarge:
+    working_directory: ~/mattermost/
+    resource_class: xlarge
+    machine:
+      image: "ubuntu-2004:202201-02"
+    environment:
+      COMPOSE_PROJECT_NAME: "circleci"
 
 jobs:
-  setup-mattermost-server:
-    working_directory: ~/mattermost/mattermost-server
+  setup-multi-product-repositories:
+    working_directory: /mnt/ramdisk/mattermost-server
```

Comment:
Are there any risks using `ramdisk`?

Reply:
The only risk is getting out of memory since it will use memory allocated to the docker imaage. But according to the job statistics, our jobs are CPU bounded and we only use about 10% of the ram for all jobs.